include/HeaderFile.h

```diff
@@ -14,6 +14,7 @@
 #include <ctype.h>
 #include <unistd.h>
 #include <limits.h>
+#include <stdint.h>
```

Comment:
intptr_t is not defined without this header file, which is a constraint violation.